app/workers/standup_meeting_group/determine_missing_standup_meetings_worker.rb

```diff
@@ -13,7 +13,11 @@ def perform(start_minutes = 0, end_minutes = 0)
 
       standup_meeting_groups =
         StandupMeetingGroup
-        .joins("LEFT OUTER JOIN standup_meetings ON standup_meeting_groups.id = standup_meetings.standup_meeting_group_id AND standup_meetings.meeting_date = '#{current_date}'")
+        .joins(
+          "LEFT OUTER JOIN standup_meetings
+            ON standup_meeting_groups.id = standup_meetings.standup_meeting_group_id
+            AND standup_meetings.meeting_date = '#{current_date}'"
+        )
```

Comment:
rubocop didn't like this line length. I couldn't pass CI without fixing it.

Reply:
oh thats on me :D. I should have just disabled for that line so ty.